### DISCUSSION TITLE
Welcome to GClh Discussions!

### DISCUSSION THREAD

**Discussion by 2Abendsegler** [2Abendsegler/GClh#1800] [Announcements]
## 👋🏼 Welcome!
  My dears, here you can
  * ask questions you’re wondering about,
  * share ideas,
  * discuss about new features,
  * ...

Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by jmsadh** [comment]
When you have a geocache logged as "found" you see it in the map in the original coordinates.
Is possible to see it in the "corrected coordinates"? 
In this way will be easer to see witch areas are free to put new hides.
Thank,  Cheers.
segulera

**Reply by capoaira** [reply]
Moin,
danke für deinen Beitrag. Das ist ein häufiger Wunsch, doch leider können wir da nichts machen. Wir habe leider keinen Zugriff auf die Kartenelememte und auch nicht auf die korrigierten Koords (Die korrigierten Koords können leider nur einzeln geladen werden, wenn der jeweilige Cache ausgewählt wird)
Diese Idee wurde schon mehrfach im offiziellen Forum von Groudspeak gefordert, leider haben die das nie umgesetzt  :(

**Reply by capoaira** [reply]
Sorry, that I wrote in german. Here my English translation ;)
I heared the reqest multiple times, also in the official forum. Unfortunately Groudspeak never do this.
We have no access to the map elements and cannot get all corrected coords at one time. So we are not able to implement this feature. (I wished we can)
